gnomad/utils/intervals.py

```diff
@@ -112,3 +114,63 @@ def _add_padding(
         return [_add_padding(i) for i in intervals]
     else:
         return _add_padding(intervals)
+
+
+def explode_intervals_to_loci(
+    obj: Union[hl.Table, hl.expr.IntervalExpression],
```

Comment:
```suggestion
    intervals: Union[hl.Table, hl.expr.IntervalExpression],
```
I think this should be named something more informative than `obj`, so I'll suggest intervals (but open to other ideas!)